src/outfit.ts

```diff
@@ -106,27 +107,40 @@ export function meatOutfit(
   if (myInebriety() > inebrietyLimit()) {
     forceEquip.push($item`Drunkula's wineglass`);
   } else if (!embezzlerUp) {
-    if (getKramcoWandererChance() > 0.05 && have($item`Kramco Sausage-o-Matic™`)) {
-      forceEquip.push($item`Kramco Sausage-o-Matic™`);
-    }
     // TODO: Fix pointer finger ring construction.
-    if (myClass() !== $class`Seal Clubber`) {
-      if (have($item`haiku katana`)) {
-        forceEquip.push($item`haiku katana`);
-      } else if (have($item`unwrapped knock-off retro superhero cape`)) {
-        if (!have($item`ice nine`)) retrieveItem($item`ice nine`);
-        forceEquip.push($item`ice nine`);
+    if (have($item`mafia pointer finger ring`)) {
+      forceEquip.push($item`mafia pointer finger ring`);
```

Comment:
I'd move the crit equipment logic into a separate function and return a Requirement, then you can merge it if doesn't conflict with the kramco or protonic checks below, that will prevent accidently equipping one item if several are required when the conflict happens.